notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Ha I will let STScI answer the first question ;-) But exactly what those limitations are would be great to answer the question I put for the intro, about why use one method or the other. I'm assuming these are only high level science products or something? I was struck by there being only 2 for such a well-studied object, so that's worth explaining for sure. </p><p> </p><p>Then you could specify what you mean by "other data products." What kind of data can we get from either of these? It looks like the one we downloaded was a spectrum? (I presume it's mostly spectra we're going to get for exoplanets, but that would be worth saying instead of "data" generically because that could mean images, integrated fluxes, etc.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>